File: src/c/katas/7kyu/alternate_case.c
char *alternateCase (char *string)
{
    char c;
    for (int i = 0; (c = string[i]); i++)
    {
        if (c >= 'A' && c <= 'Z') {
            string[i] += 32;
            continue;
        }
      
        if (c >= 'a' && c <= 'z') {
            string[i] -= 32;
            continue;
        }
    }

    return string;
}

File: src/python/katas/py7kyu/alternate_case.py
"""Kata url: https://www.codewars.com/kata/57a62154cf1fa5b25200031e."""
alternate_case = str.swapcase


def test_alternate_case():
    assert alternate_case("ABC"), "abc"
    assert alternate_case(""), ""
    assert alternate_case(" "), " "
    assert alternate_case("Hello World"), "hELLO wORLD"
    assert alternate_case("cODEwARS"), "CodeWars"
    assert alternate_case(
        "i LIKE MAKING KATAS VERY MUCH"
    ) == "I like making katas very much"
